server/settings/creator_plugins.py

```diff
@@ -215,8 +215,8 @@ class TrayPublisherCreatePluginsModel(BaseSettingsModel):
                 {
                     "name": "Version",
                     "type": "number",
-                    "default": "1",
-                    "required_column": True,
+                    "default": "0",
```

Comment:
Hmm yeah. Tricky. Because technically version 0 is allowed to be a valid version.
I believe the setting for that is `ayon+settings://core/version_start_category`.

Reply:
Yeah I was inspired this change from https://github.com/ynput/ayon-traypublisher/blob/develop/client/ayon_traypublisher/plugins/create/create_csv_ingest.py#L58 to get `None` as default instead of `1`.

Reply:
Might be an edge case for now - this works fine with the default settings.